opensrp-chw/build.gradle

```diff
@@ -384,7 +384,7 @@ repositories {
 }
 
 dependencies {
-    implementation('org.smartregister:opensrp-client-chw-core:1.1.39-DTREE-25-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-chw-core:1.1.39-DTREE-26-SNAPSHOT@aar') {
```

Comment:
@issyzac - Check this library version, the current dev branch is on version 28, and there are some changes there

Reply:
Poa poa, let me update that